.env.example

```
@@ -11,4 +11,11 @@ FORCE_ISOCODE=
 # Breez SDK API Key (for Lightning Network features)
 # Get your API key at: https://sdk-doc-liquid.breez.technology/guide/getting_started.html#api-key
-BREEZ_API_KEY=
\ No newline at end of file
+#
+# One issued Breez key currently covers both the Liquid and Spark SDKs,
+# so `BREEZ_API_KEY` is read by both backends by default. To split them
+# later (e.g. if Breez issues separate keys), set `BREEZ_SPARK_API_KEY`
+# explicitly — it takes precedence over `BREEZ_API_KEY` for the Spark
+# backend. See `coincube-gui/src/app/breez_spark/config.rs`.
+BREEZ_API_KEY=
+BREEZ_SPARK_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)